npm-package/src/server.ts

```diff
@@ -1,6 +1,7 @@
 import express, { Express, NextFunction, Request, Response, ErrorRequestHandler } from 'express';
 import cors from 'cors';
 import { otelController } from './controllers/otelController';
+import { DefaultError } from '../types/server'
```

Comment:
can we try taking out the type import and see if the declaration file auto recognizes this typing?